custom_components/alexa_media/alexa_entity.py

```diff
@@ -98,6 +98,13 @@ def is_light(appliance: Dict[Text, Any]) -> bool:
         and has_capability(appliance, "Alexa.PowerController", "powerState")
     )
 
+def is_contact_sensor(appliance: Dict[Text, Any]) -> bool:
+    """Is the given appliance a contact sensor controlled locally by an Echo."""
+    return (
+        is_local(appliance)
+        and "CONTACT_SENSOR" in appliance["applianceTypes"]
```

Comment:
This function is copied from the one used in the light integration, and there's no check on applianceTypes there.

Reply:
Just because bad code made it in, doesn't mean we keep adding it. If you believe it won't be None, then that's fine. If you're wrong, it'll just break people's integration.

Reply:
I have no idea if applianceTypes could be None or not (and I don't know how to find this information), I just trusted the existing implementation. Anyway, as you are my guru here and there's no reason to argue about a null check, I just added a safe dict get of that key in both my function and in the existing `is_light` function.

Reply:
Great thanks.